feature/announcement/src/main/java/io/github/droidkaigi/confsched2022/feature/announcement/Announcements.kt

```diff
@@ -0,0 +1,252 @@
+import io.github.droidkaigi.confsched2022.designsystem.theme.KaigiColors
+import io.github.droidkaigi.confsched2022.designsystem.theme.KaigiTheme
+import io.github.droidkaigi.confsched2022.model.AnnouncementsByDate
+import io.github.droidkaigi.confsched2022.model.fakes
+import io.github.droidkaigi.confsched2022.strings.Strings
+import io.github.droidkaigi.confsched2022.ui.UiLoadState.Error
+import io.github.droidkaigi.confsched2022.ui.UiLoadState.Loading
+import io.github.droidkaigi.confsched2022.ui.UiLoadState.Success
+import kotlinx.collections.immutable.PersistentList
+import kotlinx.datetime.Clock
+import kotlinx.datetime.DateTimeUnit
+import kotlinx.datetime.LocalDate
+import kotlinx.datetime.TimeZone
+import kotlinx.datetime.minus
+import kotlinx.datetime.todayIn
+
+@Composable
+fun AnnouncementsScreenRoot(
+    viewModel: AnnouncementsViewModel = hiltViewModel(),
+    showNavigationIcon: Boolean = true,
+    onNavigationIconClick: () -> Unit,
+) {
+    val uiModel by viewModel.uiModel
+    Announcements(
+        uiModel = uiModel,
+        showNavigationIcon = showNavigationIcon,
+        onNavigationIconClick = onNavigationIconClick
+    )
+}
+
+@Composable
+fun Announcements(
+    uiModel: AnnouncementsUiModel,
+    showNavigationIcon: Boolean,
+    modifier: Modifier = Modifier,
+    onNavigationIconClick: () -> Unit,
+) {
+    KaigiScaffold(
+        modifier = modifier,
+        topBar = {
+            KaigiTopAppBar(
+                showNavigationIcon = showNavigationIcon,
+                onNavigationIconClick = onNavigationIconClick,
+                title = {
+                    Text(
+                        text = stringResource(Strings.announcement_top_app_bar_title),
+                    )
+                },
+            )
+        },
+    ) { innerPadding ->
+        when (uiModel.state) {
+            is Error -> {
+                uiModel.state.value?.printStackTrace()
+                TODO()
+            }
+            is Success -> {
+                if (uiModel.state.value.isNotEmpty()) {
+                    AnnouncementContentList(
+                        announcements = uiModel.state.value,
+                        innerPadding = innerPadding,
+                    )
+                } else {
+                    EmptyBody()
+                }
+            }
+            Loading -> {
+                FullScreenLoading()
+            }
+        }
+    }
+}
+
+@OptIn(ExperimentalFoundationApi::class)
+@Composable
+fun AnnouncementContentList(
+    announcements: PersistentList<AnnouncementsByDate>,
+    innerPadding: PaddingValues,
+    modifier: Modifier = Modifier,
+) {
+    LazyColumn(
+        modifier = Modifier.padding(horizontal = 16.dp),
+        contentPadding = innerPadding,
+    ) {
+        announcements.forEach {
+            // TODO: Need to fix stickyHeader to make it work.
+            stickyHeader {
+                AnnouncementsHeader(dayString = it.publishedAt.convertString())
+            }
```

Comment:
stickyHeader does not work.🥺
I am using TODO because I could not solve the problem myself.